kdc/bx509d.8

```diff
@@ -75,44 +89,137 @@ Print version.
 .Xc
 Expected audience(s) of bearer tokens (i.e., acceptor name).
 .It Xo
+.Fl Fl allow-GET
+.Xc
+If given, then HTTP GET will be allowed for the various
+end-points other than
+.Ar /health .
+Otherwise only HEAD and POST will be allowed.
+By default GETs are allowed, but this will change soon.
+.It Xo
+.Fl Fl no-allow-GET
+.Xc
+If given then HTTP GETs will be rejected for the various
+end-points other than
+.Ar /health .
+.It Xo
+.Fl Fl GET-with-csrf-token
+.Xc
+If given, then HTTP GET will be allowed for the various
+end-points other than
+.Ar /health ,
+but only with a CSRF token in a
+.Ar X-CSRF-Token
+header.
+If a CSRF token is not given, then a 403 response will be
+generated with a CSRF token in the response
+.Ar X-CSRF-Token
+header.
+.It Xo
+.Fl Fl api-header= Ns Ar HEADER-NAME
+.Xc
+If given, then all requests other than to the
+.Ar /health
+service must have the given request
+.Ar HEADER-NAME
+set (the value is irrelevant).
+.It Xo
+.Fl Fl csrf-key-file= Ns Ar FILE
+.Xc
+If given, this file must contain a 16 byte binary key for keying
+the HMAC used in CSRF token construction.
+.It Xo
```

Comment:
If you decide to stick with the current option names you should probably add a brief note to each of these that it is an anti-CSRF feature. A naive user is probably otherwise going to be rather confused what something like --api-header would be used for.